lab/src/Microsoft.ServiceModel.Syndication/src/Rss20Constants.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Microsoft.ServiceModel.Syndication
 {
-    internal static class Rss20Constants
+    public static class Rss20Constants
     {
```

Comment:
This isn't an exhaustive list as you can use atom elements in an RSS feed. Also the namespace isn't here anywhere. Make this internal again and if we want to make the xml names public, we'll need to think discuss what the API will look like a bit more.

---
In reply to: [126003869](https://github.com/dotnet/wcf/pull/2028#discussion_r126003869) [](ancestors = 126003869,125780076)